File: PHP/core/week-1/c-programs/programs/7.php
<?php
    $n = 10;
    $a = 0;
    $b = 1;

    echo "Fibonacci Series up to $n terms: ";

    for ($i = 1; $i <= $n; $i++) {
        echo $a . ", ";
        $temp = $a + $b;
        $a = $b;
        $b = $temp;
    }
?>

File: One Task/Task 1/c programming/7.c
// Fibonacci Series

#include <stdio.h>

int main() {
    int a = 0, b = 1, c = 0, n = 5;

    printf("Fibonacci Series: ");
    if(n == 1) {
        printf("%d", a);
    }else if (n == 2) {
        printf("%d, %d", a, b);
    }else if (n > 2) {
        printf("%d, %d", a, b);
        for(int i=1; i<=n-2; i++) {
            c = a + b;
            printf(", %d", c);
            b = a;
            a = c;
        }
    }
    return 0;
}